bindings/python/pymongoarrow/api.py

```diff
@@ -336,6 +337,10 @@ def write(collection, tabular):
             collection.insert_many(cur_batch)
         except BulkWriteError as bwe:
             raise ArrowWriteError(_transform_bwe(dict(bwe.details), cur_offset)) from bwe
```

Comment:
A few comments for consistency:

1. _transform_bwe should only include writeErrors+writeConcernErrors+nInserted and not the other fields because write() only runs insert.
2. This is unrelated to the current PR but can we rename results["insertedCount"] to just inserted_count = 0? Then we can have ArrowWriteResult accept a named parameter inserted_count. Is there any utility for making "results" a dictionary?

Reply:
1. Answered below.
2. I think the primary utility of making results a dictionary is that there is one consistent api for accessing all of the elements. You can also do things like iterate over all the fields more easily (like if you were trying to compare two results).

Reply:
Fair enough.